CHANGELOG.md

```diff
@@ -1,33 +1,61 @@
-## 1.0.6 (December 6, 2021)
```

Comment:
@anders-schwartz-ya here's one more look at the updated change log. I saw an example out there that uses the added, changed, and fixed sections and thought it was pretty good.